x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/MachineLearningFeatureSetUsage.java

```diff
@@ -57,6 +67,7 @@ protected void innerXContent(XContentBuilder builder, Params params) throws IOEx
         if (datafeedsUsage != null) {
             builder.field(DATAFEEDS_FIELD, datafeedsUsage);
         }
+        builder.field(NODE_COUNT, nodeCount);
```

Comment:
Going with the approach of not printing "unknown", this can be:

```
        if (nodeCount >= 0) {
            builder.field(NODE_COUNT, nodeCount);
        }
```